parquet/src/test/scala/magnolify/parquet/ParquetTypeSuite.scala

```diff
@@ -193,6 +196,8 @@ class ParquetTypeSuite extends MagnolifySuite {
 
 case class Unsafe(c: Char)
 case class ParquetTypes(b: Byte, s: Short, ba: Array[Byte])
+case class MapPrimitive(strMap: Map[String, Int])
```

Comment:
Can we also test a nullable value type, since that's allowed in the Parquet spec?

Reply:
gotcha. My implementation contained a bug. Thanks!